epidose/device/util.sh

```diff
@@ -129,6 +136,50 @@ run_python()
   "$PYTHON" "$SCRIPT_DIR"/"$prog".py $DEBUG_FLAG $VERBOSE_FLAG "$@"
 }
 
+# Check the validity of the cuckoo filter
+# preconditions: none 
+# Returns the time required to sleep (in seconds)
```

Comment:
I do not see a return value.  You set a global variable, which is confusing. Please rearrange to actually return a value.  It's probably best to output the value on stdout. Document that and modify the function's name to reflect its behavior.

Reply:
I have set local variables now. Also, I have tried to actually return the values `(return $to_sleep)` and read the value as `to_sleep=$?`, but every time the script was failing if the value was not zero. Therefore, I printed as you said above and created a local variable to reading (from where the function was invoked). I made also some changes in the documentation. Let me know if this is what you had in mind.